permissions/plugin-qualys-api-security.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "qualys-api-security"
+github: "jenkinsci/qualys-api-security-plugin"
+paths:
+	- "com/qualys/plugins/qualys-api-security-plugin"
```

Comment:
Looks like you have invalid whitespace in your yaml.

Reply:
There are tabs, which, I guess need to be removed.

Reply:
Tabs are removed, and a build has passed after that.